pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
fyi the ci is complaining about this file, we may need to add it to the root MANIFEST:
```
include pytest.ini
```